aggregator_unit_test.go

```diff
@@ -0,0 +1,30 @@
+package main
```

Comment:
Usually these files are called `test_{filename}.go` (`test_aggregator.go` in this case)

Reply:
@doron-cohen test files should end with `_test.go`, otherwise `go test` will not find them. See this article: https://medium.com/rungo/unit-testing-made-easy-in-go-25077669318

Reply:
You are right.